cl/scrapers/management/commands/scrape_harvard_ia.py

```diff
@@ -100,7 +109,8 @@ def get_from_ia(options: OptionsType) -> None:
     """
     ia_session = create_ia_session()
     volumes_ids_for_reporter = fetch_ia_volumes(ia_session, options)
-
+    print(volumes_ids_for_reporter)
+    return
```

Comment:
Realizing this must not be ready for review yet. My bad. Anyway...

Reply:
always happy to have your second opinion - anytime